### ISSUE TITLE
Inconsistant tags

### ISSUE THREAD

**Issue by Helioux** [Noaaan/MythicMetals#183]
noticed that some metals are tagged with ingot and the like properly but many exclusively have "#mythicmetals:ect" or just "#material:ingot/nugget/ore" tags, this leads to some funky interactions with other mods such as create, and I would love a fix. I'm using mod version 0.17.3 on 1.19.2 for fabric.

**Comment by Helioux** [comment]
this also includes the armors and weapons added by the mod, i guess i would just like to see properly assigned vanilla tags as it greatly contributes to compatibility.

**Comment by Noaaan** [comment]
Could you provide some more meaningful/direct examples of the vanilla tags you mean here? I have been playtesting with Create on 1.19.2 for a while now and had no issues related to tags. In fact via the Alloy Forgery we support most of their recipes. 

For some clarification, the Fabric ecosystem uses convention tags, which are all in the `#c` namespace, for example `#c:adamantite_ores`. In my case these tags nest their `#mythicmetals` equivalent. All tools should be present in the main data directory or under the generated data:
https://github.com/Noaaan/MythicMetals/tree/1.20/src/main/resources/data
https://github.com/Noaaan/MythicMetals/tree/1.20/src/main/generated/data/

**Comment by Noaaan** [comment]
Closing as stale

**Comment by Helioux** [comment]
None of the ingots have the vanilla tag #C:ingots, nuggets not tagged #C:nuggets

**Comment by Noaaan** [comment]
That is not a vanilla tag, thats a convention tag, which is a part of the modding/Fabric ecospace. I do see the point of this being useful for Create item filters, thats valid. 
In the future please provide some examples first so I don't have to guess what you are suggesting

**Comment by Noaaan** [comment]
Implemented the suggested tags, thanks for the issue. Feel free to suggest more in the future.

**Comment by unilock** [comment]
Would it be possible to add `#c:<material>_nuggets` tags as well? This is especially relevant for copper nuggets, which are also added by several other mods.

**Comment by Noaaan** [comment]
Maybe. I did originally consider it, but it is somewhat problematic as you are able to disable all nuggets in the config. You cannot optionally load entries in tag to my knowledge, so supplying raw data (JSON) is very annoying to deal with (it produces a ton of errors if the items aren't present).
I could solve this with tag injection, but I do not think it is the prettiest solution. Mainly since I would be providing a much higher quantity of tags that aren't super-visible to modders. They would be visible in-game or via third party tools, which would be good.
I will deliberate this suggestion a bit, and I might implement it for upcoming versions

**Comment by unilock** [comment]
> You cannot optionally load entries in tag to my knowledge...

Would that not be accomplished by the following?

```json
{
  "replace": false,
  "values": [
    {
      "id": "mythicmetals:copper_nugget",
      "required": false
    }
  ]
}
```

**Comment by unilock** [comment]
Temporary commit to add datagen for everything: https://github.com/unilock/MythicMetals/commit/eb05e047126f2c769f6b4cd3cd66209941f08881

It doesn't currently work due to registry stuff(?), but I'm in a rush atm so I'll have to look into it later.

**Comment by Noaaan** [comment]
Yeah nah I completely forgot this was a thing. Thank you for the suggestion.
Feel free to make a PR, but I will try to have this fixed by this weekend/next week, as I am almost done with the 1.20.6 port. After that I want to get the german lang merged into 1.20.1, release it, and then port that. Currently busy with moving, so it's a bit hectic
